tests/110-test-fail-if-no-thinpool.sh

```diff
@@ -0,0 +1,36 @@
+source $SRCDIR/libtest.sh
+
+test_fail-if-no-thinpool() {
+  local devs dev
+  local devlinks devlink
+  local test_status=1
+  local testname=`basename "$0"`
+  local vg_name="dss-test-foo"
+  local infile=${WORKDIR}/container-storage-setup
+  local outfile=${WORKDIR}/container-storage
+
+  # Error out if any pre-existing volume group vg named dss-test-foo
+  if vg_exists "$vg_name"; then
+    echo "ERROR: $testname: Volume group $vg_name already exists." >> $LOGS
+    return $test_status
+  fi
+
+  cat << EOF > $infile
+DEVS="$devs"
+VG=$vg_name
+EOF
+  # Run docker-storage-setup
+  $DSSBIN $infile $outfile >> $LOGS 2>&1
+
+  # Test failed.
+  if [ $? -ne 0 ]; then
+      test_status=0
```

Comment:
We don't currently check error output since we are redirecting to the logs.  I guess we could output to a temp log file and then cat that output into the logs after checking?